### PR TITLE
Docs Sidebar fix: superfluid staking

### PR DESCRIPTION
**Pull request by xBalbinus**
Just fixing an issue with multiple #title's not causing the docs page's sidebar to render correctly.

### CODE REVIEW

File: x/superfluid/spec/README.md [hunk 1]
```diff
@@ -6,6 +6,647 @@ parent:
 -->
 
 # Superfluid Staking
+<<<<<<< HEAD
```
Comment (ValarDragon) [inline x/superfluid/spec/README.md:9]: Merge conflict?

Comment (xBalbinus) [reply x/superfluid/spec/README.md:9]: Uhh... strange! I thought I had already accepted current version before pushing... Fixing now.

Comment (xBalbinus) [reply x/superfluid/spec/README.md]: Patched & waiting for tests to pass.

**Review by daniel-farina — APPROVED** [review]
Thank you!

**Pull request merged**